code/src/components/TaskList.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import tasks from 'reducers/tasks'
+
+const TaskItem = styled.article`
+display: flex;
+align-items: center;
+border-radius: 10px;
+padding: 3px;
+margin: 1rem 1.5rem;
+min-height: 4rem;
+position: relative;
+word-break: break-word;
+overflow-wrap: wrap;
+background-color: white;
+
+@media (min-width: 678px) {
+    margin: 1rem 3rem;
+  }
+
+  @media (min-width: 1024px) {
+    margin: 1rem 5rem;
+  }
+`
+const TaskText = styled.h2`
+max-width: 70%;
+font-size: 20px;
+color: ${(props) => (props.completed ? `#b0b0b0` : `#222221`)};
```

Comment:
Love the conditional rendering using the complete from state. Super nice! The color is also pleasing and there's a certain satisfaction when pressing the button and it goes solid and gray, and shrinks down a bit. Very pleasing to click on, I must admit I've been clicking up and down on it several times, it gives a very immediate sense of satisfaction, and reminds me of popping bubble wrap (which I love to do).